privacyidea/lib/tokens/webauthntoken.py

```diff
@@ -0,0 +1,1195 @@
+                                  "Note: You will want to increase the ChallengeValidityTime along with this. "
+                                  "Default: 60")
+                    },
+                    WEBAUTHNACTION.USER_VERIFICATION_REQUIREMENT: {
+                        'type': 'str',
+                        'desc': _("Whether the user's identity should be verified when authenticating with a WebAuthn "
+                                  "token. Default: preferred (verify the user if supported by the token)"),
+                        'value': [
+                            'required',
+                            'preferred',
+                            'discouraged'
+                        ]
+                    },
+                    ACTION.CHALLENGETEXT: {
+                        'type': 'str',
+                        'desc': _('Use an alternate challenge text for telling the '
+                                  'user to confirm with his WebAuthn device')
+                    }
+                },
+                SCOPE.AUTHZ: {
+                    WEBAUTHNACTION.AUTHENTICATOR_SELECTION_LIST: {
+                        'type': 'str',
+                        'desc': _("A list of WebAuthn authenticators acceptable for authorization, given as"
+                                  "a space-separated list of AAGUIDs. Per default all authenticators are acceptable.")
+                    },
+                    WEBAUTHNACTION.REQ: {
+                        'type': 'str',
+                        'desc': _("Only the specified WebAuthn-tokens are authorized.")
+                    }
+                },
+                SCOPE.ENROLL: {
+                    WEBAUTHNACTION.RELYING_PARTY_NAME: {
+                        'type': 'str',
+                        'desc': _("A human readable name for the organization rolling out WebAuthn tokens."),
+                    },
+                    WEBAUTHNACTION.RELYING_PARTY_ID: {
+                        'type': 'str',
+                        'desc': _("A domain name that is a subset of the respective FQDNs for all the webservices the "
+                                  "users should be able to sign in to using WebAuthn tokens.")
+                    },
+                    WEBAUTHNACTION.TIMEOUT: {
+                        'type': 'int',
+                        'desc': _("The time in seconds the user has to confirm enrollment on his WebAuthn token. "
+                                  "Note: You will want to increase the ChallengeValidityTime along with this. "
+                                  "Default: 60")
+                    },
+                    WEBAUTHNACTION.AUTHENTICATOR_ATTACHMENT: {
+                        'type': 'str',
+                        'desc': _("Whether to limit roll out of WebAuthn tokens to either only platform attachments, "
+                                  "or only cross-platform attachments. Default: either"),
+                        'group': GROUP.TOKEN,
```

Comment:
Maybe we should not put this in the group.TOKEN, but in a new group WebAuthn.
There are so many policies, so that you immediately know, that you do not need to open this accordeon. ;-)